### PR TITLE
[FIX] stock_account: add total value

### PR DESCRIPTION
**Pull request by nim-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock_account/views/stock_valuation_layer_views.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
                 <field name="quantity" />
```
Comment (fmdl) [inline addons/stock_account/views/stock_valuation_layer_views.xml:48]: same things on quantity can be useful

`<field name="quantity" sum="Total Quantity"/>`

Comment (nim-odoo) [reply addons/stock_account/views/stock_valuation_layer_views.xml:48]: not really since you can have different UOM categories.

Comment (fmdl) [reply addons/stock_account/views/stock_valuation_layer_views.xml:48]: @nim-odoo you are right

(Note : it is the same issue if the currency is not the same. this case is less common).

**Comment by nim-odoo** [comment]
robodoo r+

**Pull request closed without merging**